test/minio/minio.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     component: minio
 spec:
   accessModes: ["ReadWriteOnce"]
-  storageClassName: "rook-ceph-block"
+  storageClassName: "$storage_class"
```

Comment:
Ok, regardless of makeing it configurable, should we use other storage class
(maybe hostpath?) for minio?

Using rook (or other storage) just complicate things by adding a dependency
on the storage.

Reply:
We could as long as a restart preserves whatever is in there (which hostpath should).

Reply:
Opened #737 to track this.

Reply:
I removed the commit adding configurable storage class since fixing minio
to use "standard" storage class (#744) is much simpler. We can add configrable
storage later if we have a real need.